### PR TITLE
Add SkyWalking CLI download links

### PR DESCRIPTION
**Pull request by kezhenxu94**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -77,6 +79,7 @@ If you have been familiar with SkyWalking, you could use this catalog to find th
       * [Dynamic Configuration](en/setup/backend/dynamic-config.md). Make configuration of OAP changed dynamic, from remote service or 3rd party configuration management system.
       * [Uninstrumented Gateways](en/setup/backend/uninstrumented-gateways.md). Configure gateways/proxies that are not supported by SkyWalking agent plugins, to reflect the delegation in topology graph.
     * [UI setup document](en/setup/backend/ui-setup.md).
+    * [CLI setup document](https://github.com/apache/skywalking-cli).
```
Comment (wu-sheng) [inline docs/README.md]: All these two are just menu, you should add this into the document content directly. People could read the document through the main entance, especially for the new user.

Such as https://github.com/apache/skywalking/blob/898074e7784ab89b54b896be28d947f8fafbd489/docs/en/concepts-and-designs/README.md

Comment (wu-sheng) [reply docs/README.md]: Including this
image

and  https://github.com/apache/skywalking/blob/898074e7784ab89b54b896be28d947f8fafbd489/docs/en/setup/backend/backend-ui-setup.md

Comment (wu-sheng) [reply docs/README.md]: Another reminder, we don't have a download link from the CLI repo, https://github.com/apache/skywalking-cli.

File: docs/README.md [hunk 2]
```diff
@@ -43,8 +43,10 @@ If you have been familiar with SkyWalking, you could use this catalog to find th
     * [Query in OAP](en/protocols/README.md#query-protocol). A set of query protocol provided, based on the Observability Analysis Language metrics definition.
   * UI
     * [Overview](en/concepts-and-designs/ui-overview.md).  A simple brief about SkyWalking UI.
+  * CLI (Command Line Interface)
+    * [SkyWalking CLI](https://github.com/apache/skywalking-cli) provides a command line interface to interact with SkyWalking backend (via GraphQL), for more information, [click here](https://github.com/apache/skywalking-cli).
```
Comment (wu-sheng) [inline docs/README.md]: Click here and `SkyWalking CLI` link to the same place, what is the point?

File: docs/README.md (hunk 1 above)
Comment (kezhenxu94) [reply docs/README.md]: > Another reminder, we don't have a download link from the CLI repo, https://github.com/apache/skywalking-cli.

adding

Comment (wu-sheng) [reply docs/README.md]: Still missing the update on https://github.com/apache/skywalking/blob/898074e7784ab89b54b896be28d947f8fafbd489/docs/en/setup/backend/backend-ui-setup.md, I think.

File: docs/en/setup/backend/backend-ui-setup.md [hunk 3]
```diff
@@ -37,4 +37,4 @@ data to backend. Also, jetty service supported in JSON format.
 Now, let's continue with the backend and UI setting documents.
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-ui-setup.md]: You miss this sentence :)

Comment (kezhenxu94) [reply docs/en/setup/backend/backend-ui-setup.md]: > You miss this sentence :)

😂

**Pull request merged**